src/style.js

```diff
@@ -196,3 +198,21 @@ export function translate(x, opt_y) {
 export function scale(value) {
   return `scale(${value})`;
 }
+
+
+/**
+ * Escapes an ident (ID or a class name) to be used as a CSS selector.
+ *
+ * See https://drafts.csswg.org/cssom/#serialize-an-identifier.
+ *
+ * @param {!Window} win
+ * @param {string} ident
+ * @return {string}
+ */
+export function escapeCssSelectorIdent(win, ident) {
+  if (win.CSS && win.CSS.escape) {
+    return win.CSS.escape(ident);
+  }
+  // Polyfill.
+  return cssEscape(ident);
```

Comment:
Instead, you could take the other [polyfill route](https://github.com/ampproject/amphtml/blob/master/src/polyfills/math-sign.js) and add `CSS.escape` if it's not there. That way, extensions won't have to include this code.